edenia/eden-spend-explorer
# Fix/validate if exists a new election 168

### Title

### What does this PR do?

- Resolve #168 

### Steps to test

1. Go to hapi/src/services/eden-historic-delegates.service.js and see a new function that validates if a recent election exists

#### CheckList
